hbase-mapreduce/pom.xml

```diff
@@ -255,6 +255,14 @@
         <groupId>net.revelc.code</groupId>
         <artifactId>warbucks-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>8</source>
+          <target>8</target>
```

Comment:
could you please explain this change?

Reply:
I was unaware of this. Thanks for catching it. I will remove it - must have come from editor related configs from my local dev setup.

Reply:
ah!cool thanks.